app/models/manageiq/providers/ibm_cloud/inventory/parser/vpc.rb

```diff
@@ -60,6 +60,8 @@ def instances
       instance_hardware(persister_instance, instance)
       instance_operating_system(persister_instance, instance)
       instance_network_interfaces(persister_instance, instance)
+      vm_and_template_labels(persister_instance, instance.tags.to_a)
+      vm_and_template_taggings(persister_instance, map_labels("VmIBM", instance.tags.to_a))
```

Comment:
So I believe you need to add `VmIBM` to https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/controllers/ops_controller/settings/label_tag_mapping.rb#L17-L47